airflow/utils/db.py

```diff
@@ -655,16 +655,34 @@ def create_default_connections(session: Session = NEW_SESSION):
 @provide_session
 def initdb(session: Session = NEW_SESSION):
     """Initialize Airflow database."""
-    upgradedb(session=session)
-
+    from alembic import command
+    from flask import Flask
+    from flask_sqlalchemy import SQLAlchemy
+
+    from airflow.models import Base
+    from airflow.www.fab_security.sqla.models import Model
+    from airflow.www.session import AirflowDatabaseSessionInterface
+
+    def _create_flask_session_tbl():
+        flask_app = Flask(__name__)
+        flask_app.config['SQLALCHEMY_DATABASE_URI'] = conf.get('database', 'SQL_ALCHEMY_CONN')
+        db = SQLAlchemy(flask_app)
+        AirflowDatabaseSessionInterface(app=flask_app, db=db, table='session', key_prefix='')
+        db.create_all()
+
+    Base.metadata.create_all(settings.engine)
```

Comment:
Another problem:

You could do `airflow db init` on an existing DB and it would apply migrations too. This isn't possible anymore. Do we need to detect that and warn users to use `db upgrade` (but still apply migrations anyway to maintain compat?)

Reply:
Updated it to have the existing behaviour